modules/ROOT/partials/configuration/allow_unsafe_link_target.adoc

```diff
@@ -3,14 +3,15 @@
 [[allow_unsafe_link_target]]
 == `allow_unsafe_link_target`
 
-By default all links with a `target` of _{blanktarget}_ will get a `rel` attribute of `noopener`. This will disable access to the `window.opener` object from a child tab/window that will open on click. If this is not something you care about, you can disable this option, by setting it to _false_. Although *we do not recommend* you to do so unless you have other ways of securing links to external pages using target set to _{blanktarget}_.
+By default all links with a `target` of `{blanktarget}` will get a `rel` attribute of `noopener`. This will disable access to the `window.opener` object from a child tab/window that will open on click. If this is not something you care about, you can disable this option, by setting it to _false_. Although *we do not recommend* you to do so unless you have other ways of securing links to external pages using target set to `{blanktarget}`.
```

Comment:
It is an odd inconsistency of how attribute values presented in the same sentence: __blank_ is italicised, but `noopener` it presented as a code fragment. Felt like an error to me, but if this is intentional I will revert it back to italics.

> remember the goal here is to keep it the same as production

Well, we’re fixing [some of the errors](https://github.com/tinymce/tinymce-docs/pull/2352#discussion_r899547701), right? :)

Reply:
The only actual task here was to fix conversion errors. If obvious typos or errors were found then we were to include them if we wanted but anything that's potentially controversial we should just leave it as is so it can be fixed later as there's lot of issues like this.

So this seems to fall into that bucket, especially since it's still broken on 6.x, so I'd suggest just leaving it as is for now.